public/js/components/SplitBox.css

```diff
@@ -14,7 +14,7 @@
 .split-box .controlled {
   display: flex;
   overflow: auto;
-  transition: width 0.5s ease;
+  /*transition: width 0.5s ease;*/
```

Comment:
Does this fix resizing with the mouse? Because right now that doesn't work: the transition makes it not follow the mouse. I'm assuming this transition is the reason?